src/webhook_launcher/settings.py

```diff
@@ -120,6 +120,13 @@
     from django_auth_ldap.config import LDAPSearch
     import logging
 
+    AUTH_LDAP_BIND_DN = ""
+    if config.has_option('ldap', 'ldap_bind_dn'):
+        AUTH_LDAP_BIND_DN = config.get('ldap', 'ldap_bind_dn', raw=True)
```

Comment:
Why all these raw=true 's to config.get? Afaics that turns of %() interpolation, but why?